go_utils/__init__.py

```diff
@@ -94,12 +94,18 @@
 For Mosquito Habitat Mapper, there is `mhm-photo-download` and for Land Cover, there is `lc-photo-download`.
 
 ### General Command
-In order for the photo download scripts to work, you must specify an input path and output path. 
+In order for the photo download scripts to work, you must specify an input path and output path.
 For example, `mhm_photo_download "input csv path" "output directory"` will take the Mosquito Habitat Mapper data stored in the input CSV and download the photos to the output directory. If the output directory doesn't exist, it will be created.
 
 #### All Flag
 You can use the `--all` or `-a` to set all the following flags.
 
+#### Custom Photo Naming Flags
+* Use `--name_additional` or `-add` to add custom additional information to the photo names.
+* Use `--nargs_include` or `-in` to include a field from the default photo name. Can include any of the following:
+   * For **mosquito_habitat_mapper** -- Accepted Included Names include: `url_type`, `watersource`, `latitude`, `longitude`, `date_str`, `mhm_id`, `classification`
+   * For **land_cover** -- Accepted Included Names include: `direction`, `latitude`, `longitude`, `date_str`, `lc_id`
```

Comment:
It might be nice to explain what each of these mean (even if some are self explanatory from the name).